### ISSUE TITLE
Safari doesn't render the drop down arrows properly 

### ISSUE THREAD

**Issue by rcap107** [rcap107/scikit-learn#4]
### Describe the bug

When on Safari, the arrow labels in the drop-down menus are not rendered properly, and are instead replaced with other unicode characters.

### Steps/Code to Reproduce

```
from sklearn.preprocessing import StandardScaler
from sklearn.pipeline import make_pipeline
import numpy as np

pipe = make_pipeline(StandardScaler())

X = np.random.random((1, 10))

repr = pipe._repr_html_()
with open("pipeline_repr.html", "w") as repr_file:
    repr_file.write(repr)

```

### Expected Results

<img width="181" alt="image" src="https://github.com/rcap107/scikit-learn/assets/7548232/a8c04f38-cd9d-4d29-8517-42f2045beb77">

### Actual Results

![image](https://github.com/rcap107/scikit-learn/assets/7548232/ddf88e69-3efe-43ec-a344-66b9d25e0817)

### Versions

```shell
System:
    python: 3.10.11 (main, May 15 2023, 19:29:30) [Clang 14.0.6 ]
executable: /Users/rcap/opt/anaconda3/envs/sklearn/bin/python
   machine: macOS-10.16-x86_64-i386-64bit

Python dependencies:
      sklearn: 1.4.dev0
          pip: 23.1.2
   setuptools: 67.8.0
        numpy: 1.24.3
        scipy: 1.10.1
       Cython: 0.29.35
       pandas: None
   matplotlib: None
       joblib: 1.2.0
threadpoolctl: 2.2.0

Built with OpenMP: False

threadpoolctl info:
       filepath: /Users/rcap/opt/anaconda3/envs/sklearn/lib/libmkl_rt.2.dylib
         prefix: libmkl_rt
       user_api: blas
   internal_api: mkl
        version: 2023.1-Product
    num_threads: 8
threading_layer: intel

       filepath: /Users/rcap/opt/anaconda3/envs/sklearn/lib/libiomp5.dylib
         prefix: libiomp
       user_api: openmp
   internal_api: openmp
        version: None
    num_threads: 8
```
```
